src/ui/public/index_patterns/__tests__/_index_pattern.js

```diff
@@ -405,12 +404,14 @@ describe('index pattern', function () {
 
   describe('#isWildcard()', function () {
     it('returns true if id has an *', function () {
-      indexPattern.id = 'foo*';
-      expect(indexPattern.isWildcard()).to.be(true);
+      return create('foo*').then(function (pattern) {
```

Comment:
I see this is done elsewhere in the file, so I'm not going to block on it, but in general we probably shouldn't be running assertions inside a promise resolution.  The problem with this approach is that if the promise is rejected or simply never resolves, the assertion will never run, which means this test won't fail even though a very unexpected thing just occurred.